### PR TITLE
Added StorageAccess to appendix 3

### LINKED ISSUE #232: feat: derive StorageAccess
We can now use `derive(StorageAccess)` for user-defined structs.
This small addition goes into [Appendix 3](https://cairo-book.github.io/appendix-03-derivable-traits.html)

### PR DESCRIPTION
**Pull request by ametel01**
closes #232

### CODE REVIEW

File: src/appendix-02-operators-and-symbols.md
Comment (enitrat) [inline src/appendix-02-operators-and-symbols.md:1]: Formatting broke here

**Review by enitrat — CHANGES_REQUESTED** [review]

File: src/appendix-03-derivable-traits.md [hunk 1]
```diff
@@ -148,6 +148,27 @@ When moving out of scope, variables need to be moved first. This is where the `D
...
+Example:
+
+```rust
+#[derive(Drop, storage_access::StorageAccess)]
+struct A {
+    item_one: felt252,
+    item_two: felt252,
+}
+
+#[storage]
```
Comment (enitrat) [inline src/appendix-03-derivable-traits.md:164]: The example seems odd here. I don't think it's valid even with the new syntax.

Comment (ametel01) [reply src/appendix-03-derivable-traits.md:164]: Could you elaborate more? Indeed the compiler does not complain about custom structs not deriving StorageAccess. For your reference this is the PR i referred to:
https://github.com/starkware-libs/cairo/pull/3062

Comment (enitrat) [reply src/appendix-03-derivable-traits.md:164]: Sorry, what I meant here was targeted to the `#[storage]` struct being outside of a contract - ideally, we want every code snippet to be compilable. That means that you should wrap this struct inside a contract, as a standalone storage struct doesn't make a lot of sense.

**Review by enitrat — APPROVED** [review]
LGTM :100: Thanks @ametel01

**Pull request merged**